R/remove.missing.levels.lm.R

```diff
@@ -0,0 +1,77 @@
+#' @title remove.missing.levels.lm
```

Comment:
The name is misleading. The function doesn't actually remove anything. Also, why is there an `.lm`?